dspace-xmlui/src/main/java/org/dspace/app/xmlui/aspect/administrative/collection/SetupCollectionHarvestingForm.java

```diff
@@ -185,19 +185,13 @@ public void addBody(Body body) throws WingException, SQLException, AuthorizeExce
 	    }
 	    
 	    settings.addLabel(T_label_setid);
-        Composite oaiSetComp = settings.addItem().addComposite("oai-set-comp");
-        Radio oaiSetSettingRadio = oaiSetComp.addRadio("oai-set-setting");
-        oaiSetSettingRadio.addOption("all".equals(oaiSetIdValue) || oaiSetIdValue == null, "all", "All sets");
-        oaiSetSettingRadio.addOption(!"all".equals(oaiSetIdValue) && oaiSetIdValue != null, "specific", "Specific sets");
+            //Composite oaiSetComp = settings.addItem().addComposite("oai-set-comp");
+            Radio oaiSetSettingRadio = settings.addItem().addRadio("oai-set-setting");
+            oaiSetSettingRadio.addOption("all".equals(oaiSetIdValue) || oaiSetIdValue == null, "all", "All sets");
+            oaiSetSettingRadio.addOption(!"all".equals(oaiSetIdValue) && oaiSetIdValue != null, "specific", "Specific sets");
 
-        Text oaiSetId = oaiSetComp.addText("oai_setid");
-	    oaiSetId.setSize(40);
-        if(!"all".equals(oaiSetIdValue) && oaiSetIdValue != null)
-        {
-            oaiSetId.setValue(oaiSetIdValue);
-        }
```

Comment:
In my hurry to recreate this PR, it looks like I merged changes incorrectly.  Thanks for catching this.